mmaction/datasets/pipelines/loading.py

```diff
@@ -672,6 +672,154 @@ class PyAVDecode(object):
             thread processing. Default: False.
     """
 
+    def __init__(self, multi_thread=False):
+        self.multi_thread = multi_thread
+
+    def parse_vectors(self, mv, vectors, height, width):
+        """Parse the returned vector."""
+        (w, h, src_x, src_y, dst_x,
+         dst_y) = (vectors['w'], vectors['h'], vectors['src_x'],
+                   vectors['src_y'], vectors['dst_x'], vectors['dst_y'])
```

Comment:
bracket is redundant

Reply:
It's because this line is too long, use bracket to change a line.